### ISSUE TITLE
Progress is not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#92929]
Our workbench progress reporting is not accessible. 
We report progress in views, globaly and in notificatinos, so this should be handled by the `progressService`. One example where we show progress is for long running search queries in the search viewlet.

Using `aria.alerts` for progress is imho too aggressive, and when I check what NVDA does during their installation they just use beep sounds, they become faster and faster giving you a sense of progress. Unfortunetly I do not think there is sound api from electron that we could use @deepak1556 

fyi @egamma @joanmarie @ajborka  @jvesouza @zersiax for potential ideas and how other apps report progress in an accessible way

**Comment by zersiax** [comment]
@isidorn can you tell me a bit more about what happens visually, and how long these indications are generally visible?

**Comment by isidorn** [comment]
@zersiax visualy is shown as a blue infinite line on top of the view. The line gives the effect that something is in progress. They are generaly visible depends on how long the task at hand is. Some are for half a second, while some might be for a minute (though those should be rare).

**Comment by deepak1556** [comment]
@isidorn there is one https://github.com/electron/electron/blob/master/docs/api/shell.md#shellbeep but I wouldn't suggest to use it for any accessibility stuff, also we would still need to solve for the web client, having a generic solution would be better here.

I tried loading `youtube.com` in chrome browser which has a progress load and the screen reader reports the percentage of load. If we use aria-progress, maybe it would be a decent and less intrusive experience ? Like notifying `Something has started loading`, `then indicating the percentage` and then `Something has finished loading, advice to continue interaction` ?

**Comment by isidorn** [comment]
@deepak1556 I think your idea is worth experimenting with. Though I am worried that it would bring too much noise. Let's wait for users to request this before doing anything - also most of our progress is quite fast and we should be fine for now.

**Comment by zersiax** [comment]
That's basically my thought. If this progress is only visible for a few seconds at most, I'd say leave it alone for now. I am going to, perhaps slightly naively, assume that users will understand that doing an operation may take a second or two. For longer operations, this is slightly trickier.
At that point, perhaps firing off a "loading..." every 10 or so seconds might be a good compromise, any quicker sounds spammy to me but I think that is a very personal preference. I'd say let's keep aware this is here, but leave it as is for now.

**Comment by isidorn** [comment]
@zersiax thanks for feedback. It seems we are on the same page.

**Comment by isidorn** [comment]
Leaving the progress as it is, will not change, thus closing
